examples/spark/README.md

```diff
@@ -228,9 +255,21 @@ zeppelin-controller-ja09s   1/1       Running   0          53s
 Now you have two choices, depending on your predilections. You can do something
 graphical with the Spark cluster, or you can stay in the CLI.
 
+For both choices, we will be working with this Python snippet:
```

Comment:
I think what would really help is an example that is actually graphical, but I don't know enough about spark to cook up a cloud-agnostic example that'll show a graph or something.